kuma/users/tests/test_stripe_subscription.py

```diff
@@ -157,7 +160,7 @@ def test_stripe_payment_succeeded_sends_invoice_mail(mock1, client):
     payment_email = mail.outbox[0]
     assert payment_email.to == [testuser.email]
     assert "manage monthly subscriptions" in payment_email.body
-    assert "invoice" in payment_email.subject
+    assert "Receipt" in payment_email.subject
 
```

Comment:
I think you should check for some of the language in the email too. 
For example 
```python
assert 'Invoice number test_invoice_001' in payment_email.body
assert 'You supported MDN with a $5.00 monthly subscription' in payment_email.body
```